LSP-tailwindcss.sublime-settings

```diff
@@ -2,63 +2,42 @@
 	"languages": [
 		{
 			"languageId": "html",
-			// ST3
 			"scopes": ["text.html.basic"],
 			"syntaxes": [
 				"Packages/HTML/HTML.sublime-syntax",
 				"Packages/PHP/PHP.sublime-syntax"
-			],
-			// ST4
-			"document_selector": "text.html.basic | embedding.php | text.blade",
-			"feature_selector": "text.html"
+			]
```

Comment:
You've removed the ST4 that included support for `text.blade`. Is that on purpose?

Reply:
8e2d092